uiowa/uiowa
# I2939 srop mentor display fixes

Resolves #2939 

This fixes the reported issue but also addresses the issue comments.

Also moves the thesis defense stuff to grad_core where it is supposed to be. Oops.

**Before**:
<img width="687" alt="Screen Shot 2021-01-27 at 12 14 27 PM" src="https://user-images.githubusercontent.com/4663676/106035019-39becb00-6099-11eb-9001-48d5627a6924.png">

**After**:
<img width="657" alt="Screen Shot 2021-01-27 at 12 14 51 PM" src="https://user-images.githubusercontent.com/4663676/106035084-4ba06e00-6099-11eb-957d-74453f92850e.png">

## Test

- View https://grad.local.drupal.uiowa.edu/dei/srop/mentors with stage file proxy on if you don't have files synced.
- Confirm that multiple positions are comma separated on the teaser and not on the full person node.
- Go to https://internationalighn.local.drupal.uiowa.edu and http://admissions.local.drupal.uiowa.edu/people/jessica-klein and confirm multi-value fields are comma-separated (twig debug should reference a uids_base location instead of the site's core module)
